### PR TITLE
feat(netlify-plugin-update-client-apps): add netlify-plugin-update-client-apps package

### PR DESCRIPTION
**Pull request by jmcodingde**
## What

- A [local Netlify build plugin](https://docs.netlify.com/integrations/build-plugins/create-plugins/#local-plugins) that updates given Tapper client apps' `redirect_uri`s by adding the URL of the current Netlify build. Skips on subsequent runs.

## Why

- A lot of our frontend apps use the [OAuth2 Auth Code + PKCE](https://oauth.net/2/pkce/) flow to get a Tapper `access_token`. The security model of this flow is based on pre-defined `redirect_uris` as properties of the client apps.
- This plugin automatically adds the current Netlify build URL to the list of `redirect_uris` of one or two given client apps in order for authorization to work on Netlify PR previews.

## History

- This plugin was recently added to the contribute-to-checkout repo in https://github.com/laterpay/contribute-to-checkout/pull/51. 
- Let's make it a private package so that we can reuse it in other frontend repos easily and manage the code in a central location.

## Review

- Have a look at the code
- Have a look at https://github.com/laterpay/contribute-to-checkout/pull/51

### CODE REVIEW

File: packages/netlify-plugin-update-client-apps/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   REACT_APP_DEFAULT_MODE = "test"
+   REACT_APP_CLIENT_ID = "client.1d2d15a6-cc8a-49be-b23a-f362b8ada428"
+   REACT_APP_TEST_CLIENT_ID = "client.652a7a36-8156-4960-a28d-bc01776f8778"
+   #REACT_APP_SENTRY_DSN=""
+   #REACT_APP_SENTRY_ENVIRONMENT="netlify"
+   NETLIFY_BUILD_CLIENT_ID = "client.c792ae5b-7847-42a8-8a14-a5eaf38de348"
+   # NETLIFY_BUILD_CLIENT_SECRET is set via Netlify's web UI
+
+   [[plugins]]
+   package = "./.netlify/netlify-plugin-update-client-apps"
```
Comment (thiagofelix) [inline packages/netlify-plugin-update-client-apps/README.md:32]: Given this is now a private npm package, won't the plugin folder be located at `node_modules` in the consuming app?

```suggestion
   package = "./node_modules/netlify-plugin-update-client-apps"
```

**Review by thiagofelix — APPROVED** [review]
Approved with one questions. 🚀 

So as far as I understood this plugin uses the `input` to read necessary environment variable values in order to append a new `redirect_uri` in the tapper api. 

Once the new `redirect_uri` is appended the netlify preview link should become valid from sso/hydra auth perspective.

File: packages/netlify-plugin-update-client-apps/README.md (hunk 1 above)
Comment (jmcodingde) [reply packages/netlify-plugin-update-client-apps/README.md]: Good catch! According to the [Netlify docs](https://docs.netlify.com/integrations/build-plugins/#configure-settings) we should use the package name `@laterpay/netlify-plugin-update-client-apps` here. I pushed a commit with this change to this branch ✅

**Review by thiagofelix — APPROVED** [review]
👍

**Pull request merged**